src/c++/perf_analyzer/genai-perf/genai_perf/llm_inputs/llm_inputs.py

```diff
@@ -243,21 +244,39 @@ def get_generic_dataset_json(
         Dict:
             The generic dataset JSON
         """
+        if output_format in [
+            OutputFormat.OPENAI_EMBEDDINGS,
+            OutputFormat.OPENAI_RANKINGS,
+        ]:
+            if input_type != PromptSource.FILE:
+                raise GenAIPerfException(
+                    "OpenAI embeddings and rankings only supports file input."
```

Comment:
I don't think it would be intuitive to users to combine the endpoints this way. Could you take the string version of the enum and print it in the error? Also, even if we keep the OPENAI_RANKINGS in the code, we're going to want to avoid printing any messages that say OpenAI rankings, since that API does not exist. So we might need to do a hacky workaround [here](https://github.com/triton-inference-server/client/blob/264aebf656480b36f045a533bb470199184719da/src/c%2B%2B/perf_analyzer/genai-perf/genai_perf/llm_inputs/llm_inputs.py#L47) for now if we keep OpenAI in the enum name (e.g. having an if statement that just prints "rankings").